clients/extension/scripts/popup.js

```diff
@@ -234,12 +234,14 @@ async function initPopup() {
     find('header h2').textContent = siteURL.origin;
   });
 
+  const backgroundPagePromise = new Promise(resolve => chrome.runtime.getBackgroundPage(resolve));
```

Comment:
this was always a lie...the callback to `chrome.runtime.getBackgroundPage` is called with `window`, not just functions from `extension-entry.js`. We didn't tell tsc that `extension-entry.js` actually exposes its functions on `window`...but it doesn't seem worth it to do that now either. This is just a new way to lie :)